pkg/node/config_defaults.go

```diff
@@ -113,7 +115,10 @@ var TestRequesterConfig = RequesterConfigParams{
 }
 
 func getRequesterConfigParams() RequesterConfigParams {
-	if system.GetEnvironment() == system.EnvironmentTest {
+	if strings.Contains(os.Args[0], "/_test/") ||
+		strings.HasSuffix(os.Args[0], ".test") ||
+		flag.Lookup("test.v") != nil ||
+		flag.Lookup("test.run") != nil {
 		return TestRequesterConfig
 	}
```

Comment:
I moved the logic from `GetEnvironment` to here as this was the only caller of the method.